core/domain/value_generators_domain.py

```diff
@@ -16,7 +16,8 @@
 
 """Classes relating to value generators."""
 
-from __future__ import annotations
+from __future__ import absolute_import
+from __future__ import unicode_literals
```

Comment:
Remove this modification.

Reply:
On removing these imports I am getting the following error in pre-submit checks.
![Capture](https://user-images.githubusercontent.com/10340981/140342801-a0100c6d-670d-4396-a9a5-5befd5b29f62.PNG)

Reply:
You need to do this:
```suggestion
from __future__ import annotations
```

Reply:
Okay, thanks!